tests/forms_tests/field_tests/test_filefield.py

```diff
@@ -80,3 +80,34 @@ def test_disabled_has_changed(self):
 
     def test_file_picklable(self):
         self.assertIsInstance(pickle.loads(pickle.dumps(FileField())), FileField)
+
+    def test_multiple(self):
+        f = FileField(multiple=True)
+        files = [SimpleUploadedFile('file_%s' % i, ('%s' % i).encode('utf-8')) for i in range(2)]
+        self.assertEqual(files, f.clean(files))
+
+        # ensure validation is executed for all files
+        f = FileField(multiple=True)
+        files = [SimpleUploadedFile('file_%s' % i, b'') for i in range(2)]
+        with self.assertRaisesMessage(ValidationError, "The submitted file 'file_0' is empty."):
+            f.clean(files)
+
+        def test_validator(value):
+            if value.size > 10:
+                raise ValidationError('File too large.')
+
+        f = FileField(multiple=True, validators=[test_validator])
+
+        files = [
+            SimpleUploadedFile('file_1', bytes(range(100))),
+            SimpleUploadedFile('file_1', b'foo')
+        ]
+        with self.assertRaisesMessage(ValidationError, 'File too large.'):
```

Comment:
Is UX lacking here? Is there an indication of which file is too large?

Reply:
This is a custom validation, but I modified the test to be more precise.